### PR TITLE
added qbittorrent sensor documentation

### PR DESCRIPTION
**Pull request by eliseomartelli**
**Description:**
Added the documentation of the qbittorrent sensor.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#18618

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/sensor.qbittorrent.markdown [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+The `qbittorrent` platform allows you to monitor your downloads with [qBittorrent](https://www.qbittorrent.org/) from within Home Assistant and setup automations based on the information.
+
+To enable this sensor, add the following lines to your `configuration.yaml`:
+
+```yaml
+# Example configuration.yaml entry
+sensor:
+  - platform: qbittorrent
+    url: 'http://<hostname>:<port>'
+    username: "<your username>"
```
Comment (frenck) [inline source/_components/sensor.qbittorrent.markdown]:
```suggestion
    username: YOUR_USERNAME
```

File: source/_components/sensor.qbittorrent.markdown [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    password: "<your password>"
```
Comment (frenck) [inline source/_components/sensor.qbittorrent.markdown]:
```suggestion
    password: YOUR_PASSWORD
```

**Review by frenck — CHANGES_REQUESTED** [review]
Thanks for creating a documentation PR @eliseomartelli!

I've reviewed it and left 2 suggestions.
Furthermore, for a nice presentation on our documentation website, it would be awesome if you'd include a logo for qBittorrent.

Could you take a look? 👍

**Comment by eliseomartelli** [comment]
Applied your suggestion and added the logo @frenck 👍

**Review by frenck — DISMISSED** [review]

**Comment by frenck** [comment]
Thanks for addressing that @eliseomartelli 👍 
✅ Approved. Can be merged as soon as the parent PR gets merged.
⚠️ The `ha_release` might need an update before merging this one in.

**Review by fabaff — APPROVED** [review]
Thanks :bird:

**Pull request merged**